projects/dsp-ui/src/lib/search/expert-search/expert-search.component.ts

```diff
@@ -0,0 +1,117 @@
+import { Component, EventEmitter, Inject, Input, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { KnoraApiConfig } from '@dasch-swiss/dsp-js';
+import { DspApiConfigToken } from '../../core/core.module';
+import { AdvancedSearchParams, AdvancedSearchParamsService } from '../services/advanced-search-params.service';
+
+@Component({
+    selector: 'dsp-expert-search',
+    templateUrl: './expert-search.component.html',
+    styleUrls: ['./expert-search.component.scss']
+})
+export class ExpertSearchComponent implements OnInit {
+
+    /**
+     * @param gravsearchQuery Send the gravsearch query back.
+     */
+    @Output() gravsearchQuery = new EventEmitter<string>();
+
+    /**
+     * @param toggleExtendedSearchForm Trigger toggle for extended search form.
+     */
+    @Output() toggleExpertSearchForm = new EventEmitter<boolean>();
```

Comment:
`toggleExtendedSearchForm` should be `toggleExpertSearchForm`

Reply:
done in 59c1d2d